schema.sql

```diff
@@ -33,4 +33,31 @@ alter table animals add column species_id int;
 alter table animals add constraint fk_species foreign key (species_id) references species(id);
 
 alter table animals add column owners_id int;
-alter table animals add constraint fk_owners foreign key ( owners_id) references owners(id);
\ No newline at end of file
+alter table animals add constraint fk_owners foreign key ( owners_id) references owners(id);
+
+create table vet (
+    id int generated always as identity primary key not null,
+    name varchar(255) not null,
+    age int not null,
+    date_of_graduation date not null
+);
+
+-- many-to-many relationship between the tables species and vets
+drop table if exists specialties;
+create table specialties (
```

Comment:
- [ ] According to the project requirements the table name is **specializations**. Please make sure to add **specializations** instead of **specialties** 👍

Reply:
![Screenshot 2023-05-06 142350](https://user-images.githubusercontent.com/109290913/236629998-2b424ab6-ffe7-4968-9c7c-3757cdcfb0bf.png)

@codecaiine  the subsequent name in the requirements is specialities that is why I changed the entire name to specialities and not specialization. Thank you